html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label>
+    <div>
+      <input id="email" type="text" name="email" />
+    </div>
+  </form>
+  <form>
+    <label for="whyCool">Why is coding cool?</label>
+    <div>
+      <select id="whyCool" name="whyCool">
+        <option value="select">Select an option</option>
+        <option value="interesting">Because it's interesting.</option>
+        <option value="fun">Because it's fun.</option>
+        <option value="peaceful">Because it's peaceful.</option>
+      </select>
+    </div>
+  </form>
+
+  <form>
+    <label for="language">Favorite Language (choose one)</label>
```

Comment:
This is actually just general text describing the section below since it isn't labeling one specific form control element. You should use a `p` element.